PACKAGES.md

```diff
@@ -20,13 +20,13 @@ the **edge-most** channel that corresponds to the **released** versions of git-m
 | Snap                                  | ![Snap](https://snapcraft.io/git-machete/badge.svg)                                                     | [**Snap package**](https://snapcraft.io/git-machete)                                                                                                                                                                                                                                                                                                         | Ourselves                                                                                   | Our CI on `master` builds[^6]                                                                                                                                                         |                                                                                                                                                                                                                                                                                                                       |
 | [T2 SDE](https://t2sde.org/)          | ![T2 SDE package](https://repology.org/badge/version-for-repo/t2/git-machete.svg?header=)                       | [**T2 SDE package**](https://t2sde.org/packages/git-machete)                                                                                                                                                                                                                                                                                                 | T2 SDE package maintainers                                                                  | T2 SDE package maintainers                                                                                                                                                            |                                                                                                                                                                                                                                                                                                                       |
 
```

Comment:
Check packages.... if they contain a reference to `git-machete.fish` (to be renamed git-machete.completion.fish for consistency)

Reply:
Homebrew: contains for sure, to be migrated
Nixpkgs: ?